lib/src/__tests__/setup.js

```diff
@@ -4,6 +4,17 @@ const EnzymeAdapter = require('enzyme-adapter-react-16');
 // Setup enzyme's react adapter
 Enzyme.configure({ adapter: new EnzymeAdapter() });
 
+// Fix popper mount https://github.com/mui-org/material-ui/issues/15726
+document.createRange = () => ({
```

Comment:
It should no longer be required: https://github.com/mui-org/material-ui/issues/15726#issuecomment-610084883.

Reply:
But it doesn't work without :)

Reply:
Meaning you want to upgrade jsdom in a different pull request? Great call. In this case, I would suggest a simple comment: 
```suggestion
// TODO, upgrade jsdom to 16.0.0 and remove this line.
document.createRange = () => ({
```